CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,336 @@
+  and learning from the experience to avoid future harm
+1. Focusing on what is best not just for us as individuals, but for the
+  overall community in the long term
+
+**5.2** Examples of unacceptable behaviour include:
+
+1. Trolling or insulting people in any interaction (online or in person)
+1. Using aggressive, derogatory or derisive comments
+1. Making personal or political attacks
+1. Public or private harassment of any kind
+1. Publishing others' private information, such as a physical or email address, without their explicit permission
+1. Inflicting emotional distress
+1. Any conduct which could reasonably be considered inappropriate in a professional setting
+1. The use of sexualised language or imagery, and sexual attention or advances of any kind
+ 
+## 6. Diversity Statement
+
+We encourage everyone in the team to participate in all team activities and work.
+We are committed to facilitating Communities of Practice that are connected with 
+each others work. Although we will fail at times, we seek to treat everyone both as
+fairly and equally as possible. Whenever a member has made a mistake, we
+expect them to take responsibility for it. If someone has been harmed or
+offended, it is our responsibility to listen carefully and respectfully, and do
+our best to right the wrong.
+
+Although this list cannot be exhaustive, we explicitly honour diversity in age,
+gender, gender identity or expression, culture, ethnicity, language, national
+origin, political beliefs, profession, race, religion, sexual orientation,
+socioeconomic status, and technical ability. We will not tolerate discrimination
+based on any of the protected characteristics above, including participants with
+disabilities.
+
+## 7. Restorative Practice Statement and Principles
+
+Restorative practices are based on the belief that human beings are happier, more 
+cooperative and productive, and more likely to make positive changes in their behaviour 
+when those in positions of authority do things with them, rather than to them or for them. 
+The most critical function of restorative practices is restoring and building relationships. 
+People in charge must meet with those they have wronged, explore what happened, 
+and make necessary amends.
+
+## 8. Restorative Practice Principles in this Code of Conduct
+
+1. Focuses on harms rather than the intention of wrongdoers.
+2. Prioritises persons affected and the consequent needs of victims and safety in communities.
+3. Addresses obligations of the wrongdoer resulting from those harms, as well as the team/community’s obligations to both victims and wrongdoers.
+4. Creates opportunity for all individuals in this team, who have a legitimate stake in a given situation, to be heard and their concerns addressed.
+5. Seeks to right the wrongs that have been done so that victims feel safe and valued, and wrongdoers feel restored to the team/community
```

Comment:
```suggestion
5. Seeks to right the wrongs that have been done so that victims feel safe and valued, and wrongdoers feel restored to the team/community.
```